api/src/OpenTelemetry/LogAttributes.hs

```diff
@@ -0,0 +1,173 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE InstanceSigs #-}
+
+module OpenTelemetry.LogAttributes (
+  LogAttributes,
+  emptyAttributes,
+  addAttribute,
+  addAttributes,
+  getAttributes,
+  lookupAttribute,
+  AnyValue (..),
+  ToValue (..),
+
+  -- * unsafe utilities
+  unsafeLogAttributesFromListIgnoringLimits,
+  unsafeMergeLogAttributesIgnoringLimits,
+) where
+
+import Data.ByteString (ByteString)
+import Data.Data (Data)
+import qualified Data.HashMap.Strict as H
+import Data.Hashable (Hashable)
+import Data.Int (Int64)
+import Data.String (IsString (..))
+import Data.Text (Text)
+import qualified Data.Text as T
+import GHC.Generics (Generic)
+import OpenTelemetry.Attributes (AttributeLimits (..))
```

Comment:
There is no necessary difference in `AttributeLimits` between `Attributes.hs` and `LogAttributes.hs`, so they are imported instead of rewritten.